supplements/eberron-rising-from-the-last-war/class-artificer.xml

```diff
@@ -1065,20 +1048,17 @@
 	<element name="Replicate Magic Item" type="Class Feature" source="Eberron: Rising from the Last War" id="ID_WOTC_ERLW_CLASS_FEATURE_ARTIFICER_INFUSION_REPLICATE_MAGIC_ITEM">
 		<supports>Artificer Infusion</supports>
 		<description>
-			<p>Using this infusion, you replicate a particular magic item. You can learn this infusion multiple times; each time you do so, choose a magic item that you can make with it, picking from the Replicable Items tables below. A table’s title tells you the level you must be in the class to choose an item from the table.</p>
+			<p>Using this infusion, you replicate a particular magic item. You can learn this infusion multiple times; each time you do so, choose a magic item that you can make with it, picking from the Replicable Items tables below. A table’s title tells you the level you must be in the class to choose an item from the table. Alternatively, you can choose the magic item from among the common magic items in the game, not including potions or scrolls.</p>
```

Comment:
Were going to have to make elements for every single common magic item across all the books now.
Also you made the changes to the replicate able magic item table but it also needs to be removed and added to the selects.
This also has the potential to break existing characters.